pages/checkout.vue

```diff
@@ -149,11 +149,16 @@ export default {
   setup() {
     const nuxt = useNuxtApp()
     const countries = nuxt.nuxt2Context.$config.countries
+    const paymentType = nuxt.nuxt2Context.$config.paymentAction.paymentType
+    const currencyCode = nuxt.nuxt2Context.$config.paymentAction.currencyCode
```

Comment:
remove this currencyCode config, use i18n

Reply:
Done